vdirsyncer/storage/filesystem.py

```diff
@@ -22,13 +22,15 @@ class FilesystemStorage(Storage):
     storage_name = "filesystem"
     _repr_attributes = ("path",)
 
-    def __init__(self, path, fileext, encoding="utf-8", post_hook=None, **kwargs):
+    def __init__(self, path, fileext,
+                 encoding="utf-8", post_hook=None, fileignoreext=".tmp", **kwargs):
```

Comment:
Please run `black` which will handle auto-formatting everything for you :)

Reply:
Ran `black` (Personally I prefer hand-formatted code, as it is more expressive.) Thanks for the feedback!